msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md

```diff
@@ -529,19 +536,130 @@ timer.play();
 
 ---
 
+### LiveState example
+
+:::image type="content" source="../assets/images/teams-live-share/live-share-state.png" alt-text="Screenshot shows an example of Live Share state to synchronize what planet in the solar system is actively presented to the meeting.":::
+
+The `LiveState` class enables synchronizing simple application state for everyone in a meeting. `LiveState` synchronizes two values: a `state` string and and a corresponding `data` object. This allows you to build an ephemeral distributed-state machine.
+
+Here are a few examples in which `LiveState` can be used in your application:
```

Comment:
```suggestion
You can use ‘LiveState’ in your application in the following scenarios:
```

Reply:
These are just examples of scenarios and not an exhaustive list. Your wording implies that the provided scenarios are the only possible scenarios. We should not imply to developers that it can only be used for the scenarios described in this list.